.travis.yml

```diff
@@ -24,6 +26,9 @@ before_install:
     - cmake .. && make && sudo make install
     - kcov --version
     - cargo install cargo-kcov
+before_script:
+  - rustup component add clippy
```

Comment:
Is there a way to only run this for the Clippy job itself? It's not that big of a deal, because this is generally a pretty quick procedure, but we don't need it in the other jobs.

Reply:
You can remove the whole `before_script:`, and add `rustup component add clippy && cargo clippy --version && ` after `script:` in the Clippy job below

Reply:
Done.